conans/test/integration/toolchains/ios/test_using_cmake.py

```diff
@@ -64,18 +31,17 @@ def test_xcode_generator(self):
         """ Simplest approach:
             https://cmake.org/cmake/help/latest/manual/cmake-toolchains.7.html#cross-compiling-for-ios-tvos-or-watchos
         """
-        self.t.save({'conanfile.py': self._conanfile.format(generator='"Xcode"')})
+        self.t.save({'conanfile.py': conanfile_py.format(generator='"Xcode"')})
```

Comment:
Please do not change tests not related to intel.

Reply:
I reverted other tests (ios, android), now only intel tests changed/added